mqtt_cloud_pubsub_proxy/src/main/java/com/google/cloud/pubsub/proxy/gcloud/GcloudPubsub.java

```diff
@@ -95,12 +95,16 @@
   private static final String HASH_PREFIX = "sha256-";
   private static final String PREFIX = "cps-";
   private static final String ASTERISK_URLENCODE_VALUE = "%2A";
-  // TODO - read project name from config file
-  private static final String CLOUD_PUBSUB_PROJECT_NAME = "iot-cloud-pubsub";
+  private static final String GCLOUD_PUBSUB_PROJECT_ID_ENV_VARIABLE = "GCLOUD_PUBSUB_PROJECT_ID";
+  private static final String CLOUD_PUBSUB_PROJECT_ID =
+      System.getenv(GCLOUD_PUBSUB_PROJECT_ID_ENV_VARIABLE);
+  private static final String GCLOUD_PUBSUB_PROJECT_ID_NOT_SET_ERROR =
+      "Please set the" + GCLOUD_PUBSUB_PROJECT_ID_ENV_VARIABLE
+      + "environment variable to be equal to your project id";
```

Comment:
I prefer: "Please set the " + GCLOUD_PUBSUB_PROJECT_ID_ENV_VARIABLE + " environment variable to your project id" (without the "be equal")